osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -66,11 +66,15 @@ private void load(AudioManager audio, OsuColour colours)
 
         private void playSample()
         {
-            if (Clock == null || Clock.CurrentTime - lastSampleTime <= 50)
-                return;
-            lastSampleTime = Clock.CurrentTime;
-            sample.Frequency.Value = 1 + NormalizedValue * 0.2f;
-            sample.Play();
+            if (MathHelper.Clamp(
+                nub.DrawPosition.X - nub.DrawWidth / 2, 0, DrawWidth) > 0 & MathHelper.Clamp(
+                DrawWidth - nub.DrawPosition.X - nub.DrawWidth / 2, 0, DrawWidth) > 0) {

```

Comment:
This is very messy and should not be necessary. Update Framework + merge origin/master into this branch, then do playSample() inside UpdateValue.